pandas/tests/io/parser/common/test_read_errors.py

```diff
@@ -242,7 +242,7 @@ def test_bad_header_uniform_error(all_parsers):
     msg = "Expected 2 fields in line 2, saw 4"
     if parser.engine == "c":
         msg = "Could not construct index. Requested to use 1 "
-        "number of columns, but 3 left to parse."
+        # number of columns, but 3 left to parse.
```

Comment:
This needs joining with the string above it, it probably wasn't intended as a comment

Could you then also check that the test still passes please?

Reply:
Sure thing! I made the change to make it a string again, and retested the code
<img width="886" alt="image" src="https://user-images.githubusercontent.com/18587013/204201677-74016f4a-99d1-4bd8-9504-ac24956eb1ae.png">

I have made the change as part of: efe54026d6608b13160315b5f6560f3174657302